mappings/net/minecraft/loot/context/LootContext.mapping

```diff
@@ -8,6 +8,13 @@ CLASS net/minecraft/class_47 net/minecraft/loot/context/LootContext
 	FIELD field_927 suppliers Ljava/util/Set;
 	FIELD field_928 world Lnet/minecraft/class_3218;
 	FIELD field_929 drops Ljava/util/Map;
+	METHOD <init> (Ljava/util/Random;FLnet/minecraft/class_3218;Ljava/util/function/Function;Ljava/util/function/Function;Ljava/util/Map;Ljava/util/Map;)V
+		ARG 2 luck
+		ARG 3 world
+		ARG 4 supplierGetter
```

Comment:
If I recall correctly this should be `tableGetter` (it is now called loot table, feel free to rename outdated references)